.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+- package-ecosystem: mix
```

Comment:
I haven't read the doc for dependabot, but as this repo doesn't contain any `mix`/elixir code I would think the package-ecosystem is not right here

Reply:
I think the value should be `package-ecosystem: npm`

Reply:
I think the value should be `package-ecosystem: npm`

Reply:
```suggestion
- package-ecosystem: npm
```